animeal/src/Flows/Main/Modules/Home/Main/HomeViewModel.swift

```diff
@@ -86,8 +81,54 @@ final class HomeViewModel: HomeViewModelLifeCycle, HomeViewInteraction, HomeView
             let action = model.fetchFeedingAction(request: .autoCancelFeeding)
             onFeedingActionHaveBeenPrepared?(feedingActionMapper.mapFeedingAction(action))
         case .confirmCancelFeeding:
-            load()
-            model.processCancelFeeding()
+            Task {
+                do {
+                    _ = try await model.processCancelFeeding()
```

Comment:
nit: we can mark model's method as discardableResult to get rid of using `_ = `

Reply:
Done